code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -300,6 +300,67 @@
 	. = ..()
 	AddComponent(/datum/component/armor_plate)
 
+
+//Nanotrasen medical Protosuit
+
+/obj/item/clothing/suit/space/hardsuit/protosuit
+	name = "protosuit hardsuit"
+	desc = "A failed Nanotrasen experiment, intended to be a unique medical hardsuit based on stolen Syndicate samples."
+	icon_state = "protosuit"
+	hardsuit_type = "protosuit"
+	armor = list(MELEE = 30, BULLET = 15, LASER = 15, ENERGY = 25, BOMB = 30, BIO = 100, RAD = 25, FIRE = 75, ACID = 75, WOUND = 15)
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/storage/firstaid, /obj/item/healthanalyzer, /obj/item/stack/medical)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/protosuit
+
+/obj/item/clothing/head/helmet/space/hardsuit/protosuit
+	name = "protosuit hardsuit helmet"
+	desc = "A helmet, made in old failed Nanotrasen experiment, which was intended to create a medical hardsuit based of stolen Syndicate ones. You can see a small DNA-sampler on the bottom of it."
```

Comment:
The wording of this description is awkward and contains a few spelling errors, please change this.